pkg/cmd/aad/aad.go

```diff
@@ -0,0 +1,59 @@
+package aad
```

Comment:
This is a function called by the RP so I would move it into pkg/fakerp

Reply:
I still don't think the separate package is worth it. cmd/aad/aad is not production code so the extra package is mere cognitive overhead.